packages/language-server/test/fixture/enhancedDiagnostics.astro

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This needs to be in a real file now because otherwise it cannot seems to resolve `astro/jsx-runtime`, not sure why. Might be something with how fake files works in tests.